requirements.txt

```diff
@@ -1,8 +1,8 @@
 #
 # Reference : pypi.org
 #
-ansible==9.1.0 # 8.6.0 # 8.0.0 # 6.4.0 # 5.5.0 # 2.10.7
-openshift==0.13.1 # 0.12.0 
-kubernetes==26.1.0 # 24.2.0 # 23.3.0 # 12.0.1
-dnspython==2.3.0 # 2.2.1 # 2.1.0
+ansible-core==2.16.4
```

Comment:
Note for reviewers:
See PR description for reasoning behind this change. Will not cause issues for users with existing projects environments used to run 3.x.x viya4-iac-k8s releases.